ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml

```diff
@@ -49,7 +44,6 @@ tests:
     - chain: ipi-gcp-post
     pre:
     - chain: ipi-gcp-pre
-    - chain: create-infra-move-ingress-monitoring-registry
```

Comment:
I think we want to move ingress/monitoring pods to infra nodes here. Was there a reason you removed them?